src/api/github/org.ts

```diff
@@ -157,11 +157,7 @@ export class GitHubOrg {
     await this.sendGraphQuery(modifyProjectIssueFieldMutation, data);
   }
 
-  async clearProjectIssueField(
-    itemId: string,
-    projectFieldOption: string,
-    fieldId: string
-  ) {
+  async clearProjectIssueField(itemId: string, fieldId: string) {
```

Comment:
Codecov seems to think this never exercised by tests?

On further reflection: I guess you did a good thing by submitting a drive by fix here (removing an unused arg), and I'm punishing you by asking you to add a test too ("no good deed" etc etc)... So idk, I'm fine leaving this code as is, but the extra credit super duper brownie points version of this would be to break out this arg-removal into a separate PR where we also add a test to exercise this function. But I also know that's annoying and not on the critical path, so no pressure either way!

Reply:
I would love to get those super duper brownie points, but this function itself doesn't return anything and writing a test for it doesn't seem to do much but increase the code coverage numbers. Typescript already complains if an arg is passed in that isn't used, but the only reason this slipped in was that the arg was being passed in for error reporting to Sentry in the case an exception was thrown. If I were to write a test, it would be asserting that the args being passed in are being used in the graph query, which doesn't always seem accurate given that eventually we may want to include more context for error reporting.

Reply:
Makes sense!